vic/drivers/image/src/get_global_param.c

```diff
@@ -777,6 +777,9 @@ get_global_param(FILE *gp)
             else if (strcasecmp("SKIPYEAR", optstr) == 0) {
                 sscanf(cmdstr, "%*s %hu", &global_param.skipyear);
             }
+            else if (strcasecmp("COORD_DIMS_OUT", optstr) == 0) {
+                sscanf(cmdstr, "%*s %zu", &options.COORD_DIMS_OUT);
+            }
```

Comment:
I know nc_inq_var but thats not what I mean. The 1d or 2d info it still not in the domain. Do you have time tomorrow (same time as now) to discuss this on gitter?